doc/about.rst

```diff
@@ -0,0 +1,123 @@
+.. _about:
+
+##############################################################################
+About
+##############################################################################
```

Comment:
Just offering an opinion, as someone who's had to write lots of sphinx docs, I find the short headers easier to maintain and read:

```
About
#####
```